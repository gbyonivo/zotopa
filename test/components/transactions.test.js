import React from 'react';
import { create } from 'react-test-renderer';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import combineReducers from '../../src/reducers';
import Transactions from '../../src/components/transactions';

const appState = {
  transactionReducer: {
    personalDetails: {
      3: {
        amount: 10, name: 'ivo', email: '3', date: 'd'
      }
    }
  }
};

describe('Transactions', () => {
  it('should render correctly', () => {
    const actual = create(
      <Provider store={createStore(combineReducers(appState))}>
        <Transactions />
      </Provider>
    );
    expect(actual).toMatchSnapshot();
  });
});