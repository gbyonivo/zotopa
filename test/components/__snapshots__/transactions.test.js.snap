// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transactions should render correctly 1`] = `
<table
  className=" responsiveTable"
>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Email
      </th>
      <th>
        Amount (total - Â£
        0
        )
      </th>
      <th>
        Date
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
